test/src/main/java/io/strimzi/test/k8s/cluster/Kind.java

```diff
@@ -47,7 +47,7 @@ public boolean isAvailable() {
     public boolean isClusterUp() {
         List<String> cmd = Arrays.asList("kubectl", "get", "nodes", "-o", "jsonpath='{.items[*].spec.providerID}'");
         try {
-            return Exec.exec(cmd).out().startsWith("kind");
+            return Exec.exec(cmd).out().contains("kind-cluster");
```

Comment:
@see-quick @jankalinic when you were testing the Kind stuff, was the name of the node just "kind" or "kind-cluster"? I'm wondering if there is some kind of label or something on the node that would say for 100% that it's Kind cluster and not something else.

Reply:
It would be based on
```
cat <<EOF | kind create cluster --config=-
    kind: Cluster
    apiVersion: kind.x-k8s.io/v1alpha4
    nodes:
    - role: control-plane
    - role: control-plane
    - role: control-plane
    - role: worker
    - role: worker
    - role: worker
    name: kind-cluster
    containerdConfigPatches:
    - |-
     [plugins."io.containerd.grpc.v1.cri".registry]
       config_path = "/etc/containerd/certs.d"
    networking:
       ipFamily: $IP_FAMILY
```
config defined in `setup-kind.sh`

Reply:
And here you can see it is `kind-cluster`. When you run by default kind on machine I think it is named `kind`

Reply:
i was dealing with this when setting up STs and what i came up with was that i found official support for nodes to be in form of ` provider-id: "kind://{{.NodeProvider}}/{{.ClusterName}}/{{.NodeName}}"`

Reply:
it is based on https://github.com/kubernetes-sigs/kind/blob/40c81f187425254daf2bf84360a6257a278252df/pkg/cluster/internal/kubeadm/config.go

Reply:
Maybe obtaining info from apiVersion 
```
apiVersion: kind.x-k8s.io/v1alpha4
```
would make sense?

Because you can anyway (or any user can) have a different name defined there or?

Reply:
ok going to use `kind://` based on kind code.